modules/developer_manual/pages/core/apis/provisioning-api.adoc

```diff
@@ -21,6 +21,8 @@ to perform any of the functions listed above. The Provisioning
 API app is enabled by default. The base URL for all calls to the share
 API is *owncloud_base_url/ocs/v1.php/cloud*.
 
+IMPORTANT: The Provisioning API can only be accessed by *Admins* and *Sub-Admins*
```

Comment:
This is not quite true. A normal user can query their own information - https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioning-v1/getUser.feature#L92
And can change their email address... - https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioning-v1/editUser.feature#L100
And can reset their own password - https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiProvisioning-v1/resetUserPassword.feature#L67

But they cannot add, change, delete or see anything else.

I am not sure how many words, "caveat", should be written here. Because a simple statement is best anyway.

Reply:
I check this

Reply:
Oh, I don't like this API design, this is a Provisioning API and should only accessible users, that do the provisioning...
Anyways, there is a Important statement in some sections, which says:
```
The Basic Authorization HTTP header must be used to authenticate this request, using the credentials of a user who has sufficient access rights to make the request.
--
```

This Note is missing on other sections where it should be, so I guess, we should put it in the top generally

Reply:
There needs to be some API where a user can change their own user attributes, like display name, email address, password... so that clients can call it to do the job when a user wants to reset password etc.